src/main/java/com/icthh/xm/tmf/ms/communication/lep/XmCommunicationMsLepProcessingApplicationListener.java

```diff
@@ -1,34 +1,53 @@
 package com.icthh.xm.tmf.ms.communication.lep;
 
-import com.icthh.xm.commons.config.client.service.TenantConfigService;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_KEY_COMMONS;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_KEY_SERVICES;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_KEY_TEMPLATES;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_SUB_KEY_PERMISSION_SERVICE;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_SUB_KEY_SERVICE_TENANT_CONFIG_SERICE;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_SUB_KEY_TEMPLATE_KAFKA;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_SUB_KEY_TEMPLATE_LOAD_BALANCED_REST;
+import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.BINDING_SUB_KEY_TEMPLATE_REST;
+
 import com.icthh.xm.commons.lep.commons.CommonsExecutor;
 import com.icthh.xm.commons.lep.commons.CommonsService;
 import com.icthh.xm.commons.lep.spring.SpringLepProcessingApplicationListener;
 import com.icthh.xm.commons.permission.service.PermissionCheckService;
 import com.icthh.xm.commons.topic.service.KafkaTemplateService;
 import com.icthh.xm.lep.api.ScopedContext;
 import com.icthh.xm.tmf.ms.communication.rules.businesstime.BusinessTimeConfigService;
-import lombok.RequiredArgsConstructor;
+import java.util.HashMap;
+import java.util.Map;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.RestTemplate;
 
-import java.util.HashMap;
-import java.util.Map;
-
-import static com.icthh.xm.tmf.ms.communication.lep.LepXmCommunicationMsConstants.*;
-
 
-@RequiredArgsConstructor
 @Service
 public class XmCommunicationMsLepProcessingApplicationListener extends SpringLepProcessingApplicationListener {
 
     private final BusinessTimeConfigService tenantConfigService;
     private final RestTemplate restTemplate;
+    private final RestTemplate loadBalancedRestTemplate;
     private final CommonsService commonsService;
     private final PermissionCheckService permissionCheckService;
     private final KafkaTemplateService kafkaTemplateService;
 
+    public XmCommunicationMsLepProcessingApplicationListener(BusinessTimeConfigService tenantConfigService,
+                                                             RestTemplate restTemplate,
```

Comment:
should you add qualifier here?

Reply:
i this it's correct, it's rest template for external call. next parameter it's rest template for internal call between ms

Reply:
and for me better to configure lombok than add constructor only for annotation
https://projectlombok.org/features/constructor

Reply:
@SergeyPentsov Indeed, an explicit ctor is redundant, removed.